lib/python/src/bailo/helper/model.py

```diff
@@ -263,3 +282,175 @@ def __unpack_mc(self, res):
             self.model_card = res["metadata"]
         except KeyError:
             self.model_card = None
+
+
+class Experiment:
+    """Represent an experiment locally.
+
+    :param model: A Bailo model object which the experiment is being run on
+    :param raw: Raw information about the experiment runs
+    """
+
+    def __init__(
+        self,
+        model: Model,
+    ):
+        self.model = model
+        self.raw = []
+        self.temp_dirs = []
+
+    @classmethod
+    def create(
+        cls,
+        model: Model,
+    ) -> Experiment:
+        """Create an experiment locally.
+
+        :param model: A Bailo model object which the experiment is being run on
+        :return: Experiment object
+        """
```

Comment:
I think it would be good to have a code block and some extra detail in this doc string for some more clarity on what experiment tracking includes.